app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    command_list = command.split()
+
+    if command_list[1] == command_list[2] or command_list[0] != "cd":
+        print("do nothing")
+        return None
```

Comment:
By defult if you just call `return` you will get None, no need to put `return None` but anyway try not to use blank return here, make a backward condition